e2e/kots-release-install/cluster-config.yaml

```diff
@@ -38,7 +38,7 @@ spec:
           chartname: ingress-nginx/ingress-nginx
           namespace: ingress-nginx
           version: "4.8.3"
-          order: 2
+          order: 4
```

Comment:
This chart being added with order 4 almost always triggered the race condition before